### PR TITLE
Release v1.1 - update

### PR DESCRIPTION
**Pull request by ImmanuelSegol**

### CODE REVIEW

File: docs/icicle/primitives/msm.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+msm::msm(&scalars, &points, &cfg, &mut msm_results).unwrap();
+
+...
+```
+
+You simply call `msm::msm`, if `msm_results` is a single result it will run in single MSM mode, if you are expecting many results MSM will run in batch mode.
+
+### Benchmarks
+
+TODO
```
Comment (ImmanuelSegol) [inline docs/icicle/primitives/msm.md:75]: waiting to get benchmarks from @DmytroTym

File: docs/icicle/primitives/overview.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# ICICLE Primitives
+
+This section of the documentation is dedicated to the ICICLE primitives, we will cover the usage and internal details of our primitives such as hashing algorithms, MSM an NTT.
```
Comment (jeremyfelder) [inline docs/icicle/primitives/overview.md]:
```suggestion
This section of the documentation is dedicated to the ICICLE primitives, we will cover the usage and internal details of our primitives such as hashing algorithms, MSM and NTT.
```

File: docs/icicle/primitives/overview.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- [MSM](./msm)
+- NTT
+    - Forward NTT
+    - Inverse NTT
+- ECNTT
+    - Forward ECNTT
+    - Inverse NTT
+- Scalar Vector Multiplication
+- Point Vector Multiplication
```
Comment (jeremyfelder) [inline docs/icicle/primitives/overview.md]: We also support Add and Sub operations as well

File: docs/icicle/primitives/msm.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# MSM
+
+MSM (Multi scalar multiplication) is a important primitive in ZK protocols.
```
Comment (jeremyfelder) [inline docs/icicle/primitives/msm.md]:
```suggestion
MSM (Multi scalar multiplication) is an important primitive in ZK protocols.
```

File: docs/icicle/primitives/msm.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+-   Bucket accumulation 
+-   Large triangle accumulation
+
+In most scenarios you should be using Bucket accumulation as its a more parallel friendly algorithm and will yield better performance on larger MSMs. 
+
+The exception being a large batch of smaller MSMs.
+
+```rust
+...
```
Comment (jeremyfelder) [inline docs/icicle/primitives/msm.md:19]: These look like code ticks that weren't formatted correctly...maybe its just me 🤷🏻 
```suggestion
.
.
.
```

File: docs/icicle/primitives/msm.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+`is_big_triangle=true` will activate Large triangle accumulation and `is_big_triangle=false` will activate Bucket accumulation.
+
+ICICLE MSM also supports two different modes
+
+- Batch MSM
+- Single MSM
+
+Batch MSM allows you to run many MSMs with a single API call, Single MSM will launch a single MSM computation.
+
+You dont need to do anything special to toggle between batch or single MSM.
```
Comment (jeremyfelder) [inline docs/icicle/primitives/msm.md]:
```suggestion
You don't need to do anything special to toggle between batch or single MSM.
```

**Review by jeremyfelder — APPROVED** [review]

**Pull request merged**